csharp-htk/obj/Debug/netstandard2.0/csharp-htk.AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Reflection;
+
+[assembly: System.Reflection.AssemblyCompanyAttribute("csharp-htk")]
+[assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
+[assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
```

Comment:
If possible, let's use SemVer (https://semver.org/) and start off with version `0.1.0`

I'm not sure if C# requires 4 numbers; if so, go with `0.1.0.0`